picard/ui/options/plugins.py

```
@@ -231,53 +237,90 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def uninstall_plugin(self):
+        plugin = self.items[self.ui.plugins.selectedItems()[COLUMN_NAME]]
+        buttonReply = QtWidgets.QMessageBox.question(self, _("Uninstall plugin?"),
+                _("Do you really want to uninstall the plugin '%s' ?") % plugin.name,
+                QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.No, QtWidgets.QMessageBox.No)
+        if buttonReply == QtWidgets.QMessageBox.Yes:
+            self.tagger.pluginmanager.remove_plugin(plugin.module_name)
+            item = self.ui.plugins.currentItem()
+            plugin.is_uninstalled = True
+            plugin.enabled = False
+            try:
+                config.setting["enabled_plugins"].remove(plugin.module_name)
+            except:
```

Remove the bare except clause and target specific exceptions.